src/cli/serve/read_yaml_config.js

```diff
@@ -19,6 +28,10 @@ export function merge(sources) {
         return;
       }
 
+      if (isString(val)) {
+    	  val = _replaceEnvVarRefs(val);
```

Comment:
Looks like this line is mixing tabs and spaces. Also, would you mind just inlining the `_replaceEnvVarRefs()` function here or removing the leading `_` from it's name?